pyroute2/netlink/rtnl/req.py

```diff
@@ -119,6 +120,27 @@ def encap_header(self, header):
             if override_bos:
                 ret[-1]['bos'] = 1
             return {'attrs': [['MPLS_IPTUNNEL_DST', ret]]}
+        '''
+        Seg6 encap header transform. Format samples:
+
+            {'type': 'seg6',
+            'mode': 'encap',
+            'segs':'2000::5,2000::6'}
+        '''
+        if isinstance(header['type'], int) or \
+                (header['type'] in ('seg6')):
+            ret = []
```

Comment:
Btw, why it is a list here?

Reply:
I did not know very well the library so I just followed mpls code.